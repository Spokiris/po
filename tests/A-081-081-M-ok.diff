*** tests/A-081-081-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-081-081-M-ok.outhyp	2023-10-27 19:48:53.734789910 +0100
***************
*** 23,31 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 1;1|25
! 1;2|43
! 1;3|5=3;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 23,29 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: xxl.core.CutBuffer@4d7e1886
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 45,52 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 5;1|25
  5;2|43
! 5;3|5=3;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 43,53 ----
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 5;1|25
+ 
  5;2|43
! 
! 5;3|=3;2
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
