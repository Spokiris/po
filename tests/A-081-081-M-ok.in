4
3
2;1:2;3
7
6
5;1:5;3
1
5;1:5;3
0
0
