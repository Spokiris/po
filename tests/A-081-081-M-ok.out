Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 4
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 3
Especifique a gama (startline;startcol:endline;endcol): 2;1:2;3
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 7
1;1|25
1;2|43
1;3|5=3;2
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 6
Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1
Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
5;1|25
5;2|43
5;3|5=3;2
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
